test/test.sh

```diff
@@ -8,15 +8,19 @@ function ok()
 
 function fail()
 {
-	echo "  FAIL"
```

Comment:
This changes could be in another PR, since it's not related exclusively with the authorized_keys support, I think.